splinterd/src/daemon.rs

```diff
@@ -914,9 +916,38 @@ pub enum StartError {
     ThreadError(String),
 }
 
+impl Error for StartError {}
+
+impl fmt::Display for StartError {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            StartError::TransportError(msg) => write!(f, "transport returned an error: {}", msg),
+            StartError::NetworkError(msg) => write!(f, "network returned an error: {}", msg),
+            StartError::StorageError(msg) => write!(f, "unable to set up storage: {}", msg),
+            StartError::ProtocolError(msg) => {
+                write!(f, "unable to start due to a protocol error: {}", msg)
+            }
+            StartError::RestApiError(msg) => {
+                write!(f, "unable to start due to a rest api error: {}", msg)
+            }
```

Comment:
The format of these two variants differs from the others by including "unable to start due to ..."

Reply:
fixed